soc/intel/intel_adsp/ace/Kconfig.defconfig.series

```diff
@@ -60,6 +60,20 @@ config LOG_BACKEND_ADSP
 
 endif # LOG
 
+config XTENSA_MORE_SPIN_RELAX_NOPS
+	default y if SMP && MP_MAX_NUM_CPUS > 1
```

Comment:
Is this really an ACE thing?  Seems like most Xtensa SMP configs are going to be on similar interconnects and behave similarly, no?  cAVS 2.5 really doesn't want this?

Reply:
This has only been tested on ACE. If SOF wants it on CAVS, we can submit another patch.